src/sentry/sentry_metrics/querying/api.py

```diff
@@ -96,13 +122,12 @@ def _transform_meta_type(self):
 class ExecutionResult:
```

Comment:
nit: renaming this to `MetricsQueryResult` would make it more explicit about what it is.